site/concurrency-primitives.md

```diff
@@ -14,18 +15,15 @@ In the [`fs2.concurrent` package](https://github.com/functional-streams-for-scal
 
 Topic implements the publish-subscribe pattern. In the following example, `publisher` and `subscriber` are started concurrently with the publisher continuously publishing the string `"1"` to the topic and the subscriber consuming it until it received four elements, including the initial element, the string `"Topic start"`.
 
-```scala
-import cats.effect.{ConcurrentEffect, ContextShift, IO}
+```scala mdoc:silent
+import cats.effect._
+import cats.effect.unsafe.implicits.global
 import fs2.Stream
 import fs2.concurrent.Topic
 
-import scala.concurrent.ExecutionContextExecutor
-
-implicit val ec: ExecutionContextExecutor = scala.concurrent.ExecutionContext.global
-implicit val cs: ContextShift[IO] = IO.contextShift(ec)
-implicit val c: ConcurrentEffect[IO] = IO.ioConcurrentEffect
+val topic: IO[Topic[IO, String]] = Topic("Topic start")(Concurrent[IO])
```

Comment:
imho typeclass instances should never be passed around manually, except as a workaround where more than one instance exists. especially not in example code that will be widely copied.
if it works, i'd suggesst writing `Topic[IO,String](...)` instead.